coderd/authorize.go

```diff
@@ -21,9 +21,7 @@ func (api *API) Authorize(rw http.ResponseWriter, r *http.Request, action rbac.A
 	roles := httpmw.AuthorizationUserRoles(r)
 	err := api.Authorizer.ByRoleName(r.Context(), roles.ID.String(), roles.Roles, action, object.RBACObject())
 	if err != nil {
-		httpapi.Write(rw, http.StatusForbidden, httpapi.Response{
-			Message: err.Error(),
-		})
+		httpapi.Forbidden(rw)
```

Comment:
**question**: what allowed us to make this change? Does it mean that we no longer send those random `Message` errors on forbidden routes?

Reply:
@vapurrmaid for security reasons all forbidden messages should be identical. If they were different, then the different errors allow the end user to gain information.

----

There is a line of "security" vs "usability" that we will need to decide on these endpoints. As it's unhelpful by design.